docs/about/compatibility/openmp.md

```diff
@@ -472,3 +472,93 @@ A no-loop kernel is not generated if the OpenMP teams construct uses a `num_team
 #### Cross-team optimized reduction kernel generation
 
 If the OpenMP construct has a reduction clause, the compiler attempts to generate optimized code by utilizing efficient cross-team communication. New APIs for cross-team reduction are implemented in the device runtime and are automatically generated by clang.
+
+### Shared memory support
+
+Unlike other discrete GPU systems that have distinct CPU (system) and GPU memory, MI300A architecture features a single physical storage per socket which is the main memory for both CPU and GPU.
+To support the optimal use of shared memory on MI300A, the ROCm OpenMP compiler and runtime are optimized to use “zero-copy” also known as “no-copy” behavior as the default execution model where the CPU and GPU threads use the same addresses to access the same physical storage. The zero or no-copy behavior also ensures that the OpenMP map constructs do not result in device memory allocation and copies between host and device memory, as there is no physical device storage.
+
+For demonstration purposes, we are using [vec_add.cpp](#sample-program) in the examples below.
+
+To enable zero-copy behavior by default in the programs running on MI300A, follow these options:
+
+* When using `unified_shared_memory` pragma in the program:
```

Comment:
Single bullet point?

Reply:
The next bullet is at line 517.

Reply:
Should the text under each bullet point not be indented to cluster the content with the bullet?

Reply:
> Should the text under each bullet point not be indented to cluster the content with the bullet?

I tried to..Please read the older comments on this PR. Nested lists do not render well hence I removed nesting. Let me know if you know a way out.